cdap-ui/app/directives/log-viewer/log-viewer.less

```diff
@@ -32,6 +32,48 @@
 @error-color: #f65454;
 @warning-color: #f6b354;
 @selected-row-color: #e4e4e4;
+@application-primary-color: #43b82f;
+@application-secondary-color: #d15668;
+@application-completed-color: #5a84e4;
+@application-default-color: #969696;
+
+body.theme-cdap {
+  .modal.cdap-modal {
```

Comment:
I believe this will node config modal styles. We should target `node-config-modal` with another class so that the namespacing becomes unique for log-viewer. This I believe might affect config modal in hydrator.